docker/package/defaults/tezos-rollup-node.conf

```
@@ -0,0 +1,10 @@
+# vim: ft=sh
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+# shellcheck disable=SC2034
+DATA_DIR="/var/lib/tezos/.tezos-client"
+NODE_RPC_ENDPOINT="http://localhost:8732"
+ROLLUP_NODE_RPC_ENDPOINT="127.0.0.1:8472"
+MODE="observer"
```

I would change this to something more specific like:
```suggestion
ROLLUP_MODE="observer"
```